versions.json

```
@@ -1,4 +1,8 @@
 [
+    {
+        "version": "1.38.0",
+        "release_date": "2022-07-27"
```

Let's use today's date
```suggestion
        "release_date": "2022-08-01"
```